guides/common/modules/proc_adding-images-to-server.adoc

```diff
@@ -30,7 +30,18 @@ ifdef::kvm-provisioning[]
 endif::[]
 ifdef::rhv-provisioning,openstack-provisioning,gce-provisioning[. From the *Image* list, select an image from the {CRname} compute resource.]
 ifdef::azure-provisioning[]
+////
 . In the *Marketplace Image URN* field, enter a Marketplace URN. For example, *RedHat:RHEL:7-RAW:latest*. For more information, see https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage[Find Linux VM images in the Azure Marketplace with the Azure CLI].
+////
+. In the *Azure Image Name*, enter the UUID of the image.
+[NOTE]
+====
+You must add a prefix to the UUID based on the type of Azure image:
+* For custom images, add prefix `custom://`. For example, custom://image-name.
+* For shared gallery image, add prefix `gallery://`. For example, gallery://image-name.
+* For public and RHEL BYOS images, add prefix `marketplace://`. For example, marketplace://OpenLogicCentOS:7.5:latest. 
```

Comment:
Hah, this is something I added in #249 and asked your review in the [BZ#1862414](https://bugzilla.redhat.com/show_bug.cgi?id=1862414), but forgot to add your name in the BZ. Sorry, my fault. Can you please review the PR #249 first so that not collisions happen? Thank you

Reply:
Oh, ok. yeah, didn't see this earlier, there was no notification. I'll take a look at #249, but then merge conflict is bound to happen with my PR as it's touching the same .adoc files.

Reply:
Yes, you will need to rebase your branch

Reply:
@spetrosi I've resolved the conflict without having to rebase and the PR is mergeable again. it's ready for review.